main/src/library/Random.flix

```diff
@@ -59,17 +59,19 @@ mod Random {
     ///
     /// In other words, re-interprets the `Random` effect using the `NonDet` and `IO` effects.
     ///
-    pub def handle(f: a -> b \ ef): (a -> b \ (ef - Random) + {NonDet, IO}) \ {NonDet, IO} =
-        let rnd = new JRandom(); // Reuse Random instance.
```

Comment:
I've put the object inside the function such that `handle(f)` is stateless and wont interfere if invoked multiple times